site/models/buzz-entry.php

```diff
@@ -1,10 +1,11 @@
 <?php
 
-use Kirby\Cms\Page;
 use Kirby\Content\Field;
 use Kirby\Toolkit\Str;
 
-class BuzzEntryPage extends Page
+require_once __DIR__ . '/default.php';
```

Comment:
`Suggestion:` Instead of having to add it to each page model, we can add this line in the `config.php` or `site` plugin's `index.php`.